src/CommandingOfficers/AWBW/BH/Kindle.java

```diff
@@ -0,0 +1,152 @@
+package CommandingOfficers.AWBW.BH;
+
+import java.util.ArrayList;
+import CommandingOfficers.*;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.MassDamageEvent;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitModifierWithDefaults;
+import UI.UIUtils;
+import Units.Unit;
+import lombok.var;
+import Terrain.MapMaster;
+
+public class Kindle extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Kindle", UIUtils.SourceGames.AWBW, UIUtils.BH);
+      infoPages.add(new InfoPage(
+            "Kindle (AWBW)\n"
+          + "Units (even air units) gain +40% attack while on urban terrain. HQs, bases, airports, ports, cities, labs, and comtowers count as urban terrain.\n"));
+      infoPages.add(new InfoPage(new UrbanBlight(null, null),
+            "All enemy units lose -3 HP on urban terrain. Urban bonus is increased to +80%.\n"
+          + "+10 attack and defense.\n"));
+      infoPages.add(new InfoPage(new HighSociety(null, null),
+            "Urban bonus is increased to +130%, and attack for all units is increased by +3% for each of your owned urban terrain.\n"
+          + "+10 attack and defense.\n"));
```

Comment:
Your git commit says Kindle's AWBW SCOP has 10 too much urban boost.
AW3 Kindle gets +80, AWBW gets +130. 130-80 != 10. Please explain.

Reply:
This description is a copy/paste of the AWBW description, for obvious reasons.
AWBW's descriptions (when modifying a stat in a power) say "increased to", i.e. giving the new total for *that* boost.
So AWBW Kindle goes from 40 to 130 urban boost on SCOP,
If she has 3 cities, that's an additional +9 attack anywhere.
Add the generic +10/10, and she has 249 on urban and 119 attack elsewhere.

I otherwise prefer to simply give the relative boosts, with totals (which will include generic stats/etc) being optional.
DS Kindle goes from 40 to 120 urban boost on SCOP (a difference of 80).

130 -120 = 10

Reply:
Okay, so just a result of the generic boost difference between the two games. Or no, it looks like you add that separately. Did AWBW just give her an extra 10 power for no reason? I mean, I trust you did your homework, but that seems weird.

Reply:
They totally did. It really doesn't matter that much, TBH, given the enormous total in either case.

...and I made the mistake of telling them, so they might fix it. Oh, well.

Reply:
Oh. Well then. I guess we can go ahead and merge this then.